cmd/api/main.go

```diff
@@ -15,26 +16,30 @@ func main() {
 
 	r := gin.Default()
 
-	err := db.AutoMigrate(&models.Book{})
+	err := db.AutoMigrate(&book.Book{})
 	if err != nil {
 		log.Fatalf("Failed to migrate the database schema: %v", err)
 		return
 	}
-	err = db.AutoMigrate(&models.Author{})
+	err = db.AutoMigrate(&author.Author{})
 	if err != nil {
 		log.Fatalf("Failed to migrate the database schema: %v", err)
 		return
 	}
-	err = db.AutoMigrate(&models.Genre{})
+	err = db.AutoMigrate(&genre.Genre{})
 	if err != nil {
 		log.Fatalf("Failed to migrate the database schema: %v", err)
 	}
 
-	err = db.AutoMigrate(&models.Library{})
+	err = db.AutoMigrate(&library.Library{})
 	if err != nil {
 		log.Fatalf("Failed to migrate the database schema: %v", err)
 	}
 
 	router.SetupRouter(r, db)
 
-}
+	err = r.Run(":8080")
```

Comment:
Port config can also be part of a common config file. Use toml/env files for storing such config.

Reply:
Done